cri/v1alpha2/cri_utils.go

```diff
@@ -448,6 +448,36 @@ func setupSandboxFiles(sandboxRootDir string, config *runtime.PodSandboxConfig)
 	return nil
 }
 
+// setupPodNetwork sets up the network of PodSandbox and return the netnsPath of PodSandbox
+// and do nothing when networkNamespaceMode equals runtime.NamespaceMode_NODE.
+func (c *CriManager) setupPodNetwork(ctx context.Context, id string, networkNamespaceMode runtime.NamespaceMode, config *runtime.PodSandboxConfig) (string, error) {
+	var netnsPath string
+	hostNet := networkNamespaceMode == runtime.NamespaceMode_NODE
+	// If it is in host network, no need to configure the network of sandbox.
+	if !hostNet {
```

Comment:
How about 
```
if hostNet {
    return netnsPath, nil
}
```

Use `return fast` to reduce ident and make code more clear.

Reply:
exactly, and has been updated. ^_^